src/components/Input/index.js

```diff
@@ -0,0 +1,93 @@
+import {
+  FormControl,
+  FormHelperText,
+  MenuItem,
+  Select,
+  SvgIcon,
+  InputLabel,
+  Typography,
+} from "@material-ui/core";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+import { ReactComponent as ExpandMore } from "@/pesayetu/assets/icons/expand_more.svg";
+
+function ExpandMoreIcon(props) {
+  return <SvgIcon {...props} component={ExpandMore} />;
+}
+
+function Input({ label, helperText, options, selected, onChange, ...props }) {
```

Comment:
You've seriously decided to go with `Input `instead of `Select`?

Reply:
🥴 Fixed